setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="rhumba",
-    version='0.1.18',
+    version='0.2.0',
```

Comment:
Letters in that part of the version are part of the standard used by pip and friends (and they indicate "prerelease" versions that aren't installed without an explicit version number or a `--pre` flag), and my suggestion is specifically to use them for things that are never published to pypi.

(It was also just a suggestion. I'm happy with whatever you decide to do here.)